cmd/archive.go

```diff
@@ -56,7 +61,10 @@ func (c *cmdArchive) flagSet() *pflag.FlagSet {
 	flags.SortFlags = false
 	flags.AddFlagSet(optionFlagSet())
 	flags.AddFlagSet(runtimeOptionFlagSet(false))
-	flags.StringVarP(&c.archiveOut, "archive-out", "O", c.archiveOut, "archive output filename")
+	flags.StringVarP(
+		&c.archiveOut, "archive-out", "O", c.archiveOut,
+		"archive output filename. The special value - will cause the archive to be output to stdout.",
```

Comment:
Non-blocking: maybe explicitly name the value to be precise? 

```suggestion
		"archive output filename. Dash (-) is a reserved value that causes the archive to be output to stdout.",
```